src/Misc/layoutbin/en-US/strings.json

```diff
@@ -245,6 +246,13 @@
     "--username     basic authentication username",
     "--password     basic authentication password",
     "",
+    "",
+    "Arguments for enableautologon (Only on Windows):",
+    "-----------------------------------------------------------------------------",
+    "--restartnow          Used after autologon configuration is done. Recommended to restart the machine for the configured settings to take place.",
```

Comment:
Do we need a prompt for this? Is there a scenario where the user wants to enable autologon/autostartup but not disable the screensaver? Wouldn't that scenario simply be the Windows Service configuration flow instead?